### DISCUSSION TITLE
Trying compile to armhf

### DISCUSSION THREAD

**Discussion by dante6913** [xixogo5105/uiptv#5] [General]
Hi, when i 'm trying to compile, I get this error:
 Building UIPTV 0.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.196 s
[INFO] Finished at: 2024-11-24T16:01:34Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project uiptv: Could not resolve dependencies for project com.spc:uiptv:jar:0.1-SNAPSHOT: The following artifacts could not be resolved: org.openjfx:javafx-controls:jar:${javafx.platform}:21.0.2, org.openjfx:javafx-graphics:jar:${javafx.platform}:21.0.2, org.openjfx:javafx-base:jar:${javafx.platform}:21.0.2, org.openjfx:javafx-fxml:jar:${javafx.platform}:21.0.2: org.openjfx:javafx-controls:jar:${javafx.platform}:21.0.2 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> 

Any ideas?
Thanks for your great work

**Comment by xixogo5105** [comment]
i dont have this machine. can you please try this one below:
https://stackoverflow.com/questions/78542808/maven-project-fails-to-resolve-javafx-dependencies
"I face the exact same issue when using maven 3.9.7. It still works with maven 3.9.6"
